Formula/rails.rb

```diff
@@ -1,9 +1,9 @@
 class Rails < Formula
-  # cite L_Warren_2016: "https://doi.org/10.21105/joss.00116"
+  # cite Warren_2016: "https://doi.org/10.21105/joss.00116"
```

Comment:
I've reported this incorrect Crossref metadata to JOSS at https://github.com/openjournals/joss/issues/521
cc @warrenlr

Reply:
Great! Yeah I just noticed it so figured I'd fix it manually